workspaces/ui-v2/src/optic-components/diffs/contexts/SimulatedCommandContext.tsx

```diff
@@ -27,14 +27,15 @@ export function SimulatedCommandStore(props: SimulatedCommandStoreProps) {
       const simulated = await props.spectacle.fork();
       await simulated.mutate({
         query: `
-mutation X($commands: [JSON]) {
-  applyCommands(commands: $commands) {
+mutation X($commands: [JSON], $commitMessage: String) {
+  applyCommands(commands: $commands, commitMessage: $commitMessage) {
     batchCommitId
   }
 }
         `,
         variables: {
           commands: props.previewCommands,
+          commitMessage: '',
```

Comment:
I'm not entirely sure why / when this runs (seems to run on the new diff page) - but this needs to be shimmed since we aren't automatically adding a new commit message